packages/cojson/src/coValueCore/coValueCore.ts

```diff
@@ -1477,12 +1479,15 @@ export class CoValueCore {
     }
 
     this.markPending("storage");
+
+    trackPerformanceMark("loadFromStorageStart", this.id);
     node.storage.load(
       this.id,
       (data) => {
         node.syncManager.handleNewContent(data, "storage");
       },
       (found) => {
+        trackPerformanceMark("loadFromStorageEnd", this.id);
```

Comment:
We should track also if the value was found, to be able to exclude storage when presenting it to the user